nsv13/code/modules/overmap/traders_items.dm

```diff
@@ -49,6 +49,11 @@
 	qdel(inhabited_trader)
 	. = ..()
 
+/obj/structure/overmap/trader/process()
```

Comment:
Surely there's a better way to check without firing the check every process tick

Reply:
minor but fair enough. Moved to LateInitialize which won't catch admins doing stuff but works otherwise.